pkg/config/config.go

```diff
@@ -151,7 +151,14 @@ type Config struct {
 
 // Environ returns the config as a list of environment variables.
 func (c *Config) Environ() []string {
-	envs := []string{}
+	ex, err := os.Executable()
+	if err != nil {
+		ex = "soft"
+	}
```

Comment:
Looking again at this, perhaps we can compute this only once in the module `init()` instead of computing the value every time we call `Environ()`. Could you put this in a global variable `binPath`?